src/Endpoints/AppsEndpoint.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace JacobDeKeizer\Ccv\Endpoints;
+
+use JacobDeKeizer\Ccv\Models\Apps as Models;
+
+class AppsEndpoint extends BaseEndpoint
+{
+    public function allForStoreCategory(int $storeCategoryId): Models\Collection\Apps
+    {
+        $result = $this->doRequest(self::GET, "appstorecategories/{$storeCategoryId}/apps/");
```

Comment:
Can you use sprintf('appstorecategories/%s/apps/', $storeCategoryId) here like the other endpoints.

Reply:
Actually in the other endpoints I see concatenations. I guess you prefer it the other way because of Github's syntax highlighting? In PhpStorm it is correctly highlighted, so I prefer that over other methods.

Reply:
I prefer single quotes over double quotes, because it's more explicit. You're right that it's not consistent :)